### PR TITLE
fix(uniq): update example to cover Internet Explorer

### PR DESCRIPTION
**Pull request by yarnball**
I've found an issue with `_.uniq` and Internet Explorer (IE).

IE does not support `new Set(iterable)`

See screenshot from `mdn` below:
screen shot 2018-11-30 at 11 30 03 am

So, I've updated the example to be IE compatible.
```
[...new Set(array)];
```

to...
```
 array.filter((v, i, a) => a.indexOf(v) === i)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1292,7 +1290,7 @@ Produces a duplicate-free version of the array, using === to test object equalit
 
   // Native
   var array = [1, 2, 1, 4, 1, 3];
-  var result = [...new Set(array)];
+  var result = array.filter((v, i, a) => a.indexOf(v) === i);
```
Comment (stevemao) [inline README.md]: Can you create a new example instead of changing existing one?

File: README.md [hunk 2]
```diff
@@ -4,11 +4,9 @@ Lodash and Underscore are great modern JavaScript utility libraries, and they ar
 
 You are welcome to contribute with more items provided below.
 
-* If you are targeting legacy JavaScript engine with those ES5 methods, you can use [es5-shim](https://github.com/es-shims/es5-shim)
+**If you are targeting legacy JavaScript engine with those ES5 methods, you can use [es5-shim](https://github.com/es-shims/es5-shim)
 
-* Please note that, the examples used below are just showing you the native alternative of performing certain tasks. For some of the functions, Lodash provides you more options than native built-ins. This list is not a 1:1 comparison.
-
-* Please send a PR if you want to add or modify the code. No need to open an issue unless it's something big and you want to discuss.
+**Please note that, the examples used below are just showing you the native alternative of performing certain tasks. For some of the functions, Lodash provides you more options than native built-ins. This list is not a 1:1 comparison.
```
Comment (stevemao) [inline README.md]: Why you change these?

File: lib/rules/rules.json [hunk 3]
```diff
@@ -198,7 +198,7 @@
     },
     "uniq": {
         "compatible": true,
-        "alternative": "[... new Set(arr)]",
+        "alternative": "Array.prototype.filter((v, i, a) => a.indexOf(v) === i)",
```
Comment (stevemao) [inline lib/rules/rules.json]: Why? this guide is not only for IE. Supporting IE is nice to have

File: README.md [hunk 4]
```diff
@@ -1291,10 +1289,10 @@ Produces a duplicate-free version of the array, using === to test object equalit
   // output: [1, 2, 4, 3]
 
   // Native
-  var array = [1, 2, 1, 4, 1, 3];
-  var result = [...new Set(array)];
+  var array = [1, 1, 2, 3, 4, 4, 4];
+  var result = array.filter((v, i, a) => a.indexOf(v) === i);
   console.log(result)
-  // output: [1, 2, 4, 3]
+  // output: [1, 2, 3, 4]
```
Comment (stevemao) [inline README.md]: Don't change the original. Add a new one and comment "For IE"

File: README.md (hunk 2 above)
Comment (cht8687) [reply README.md]: @yarnball please check this review comments 😄

File: lib/rules/rules.json (hunk 3 above)
Comment (GeorgeWL) [reply lib/rules/rules.json]: still hasn't actually been changed after review requested

File: README.md [hunk 5]
```diff
@@ -511,11 +611,11 @@ Gets the first element or all but the first element.
   // output [2, 3]
   ```
 
-#### Browser Support
+#### Browser Support for Spread in array literals
```
Comment (GeorgeWL) [inline README.md]: why have you updated these?

File: README.md [hunk 6]
```diff
@@ -683,206 +927,76 @@ Returns an array where matching items are filtered.
...
-  1.0 ✔  |  1.5 ✔ |  9 ✔ |  ✔ |  ✔  |
+![Chrome][chrome-image] | ![Edge][edge-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
+:-: | :-: | :-: | :-: | :-: | :-: |
+  1.0 ✔  | ✔ | 1.5 ✔ |  9 ✔ |  ✔ |  ✔  |
 
 **[⬆ back to top](#quick-links)**
 
-### _.intersection
-Returns an array that is the intersection of all the arrays. Each value in the result is present in each of the arrays.
+## Collection*
```
Comment (GeorgeWL) [inline README.md]: If you want to add new methods, do them in a new request.

We try to keep it to one change of method per PR

**Pull request closed without merging**